environment/src/test/java/com/sequenceiq/environment/environment/EnvironmentStatusTest.java

```diff
@@ -33,6 +33,7 @@
                 { "UPDATE_FAILED",                          EnvironmentStatus.UPDATE_FAILED,                        false },
                 { "ENVIRONMENT_RESOURCE_ENCRYPTION_INITIALIZATION_IN_PROGRESS", EnvironmentStatus.ENVIRONMENT_RESOURCE_ENCRYPTION_INITIALIZATION_IN_PROGRESS, false},
                 { "ENVIRONMENT_RESOURCE_ENCRYPTION_DELETE_IN_PROGRESS", EnvironmentStatus.ENVIRONMENT_RESOURCE_ENCRYPTION_DELETE_IN_PROGRESS, true},
+                { "ENVIRONMENT_ENCRYPTION_RESOURCES_DELETED", EnvironmentStatus.ENVIRONMENT_ENCRYPTION_RESOURCES_DELETED, false},
```

Comment:
I suspect this status should also result in `true`; any status after the start of the env deletion flow (i.e. anything after `EnvironmentStatus.DELETE_INITIATED`) is essentially "deletion in progress" until the environment gets into the `ARCHIVED` status.

@doktoric @bergerdenes @biharitomi Please confirm.

Reply:
I think it should be true yes